modules/tinymce/src/core/test/ts/browser/options/NormalizeOptionsTest.ts

```diff
@@ -284,31 +284,31 @@ describe('browser.tinymce.core.options.NormalizeOptionsTest', () => {
     it('Merged settings forced_plugins in default override forced_plugins in user settings', () => {
       assert.deepEqual(
         NormalizeOptions.combineOptions(false, false, {}, { forced_plugins: [ 'a' ] }, { forced_plugins: [ 'b' ] }),
-        { external_plugins: {}, forced_plugins: [ 'b' ], plugins: 'a' },
+        { external_plugins: {}, forced_plugins: [ 'a' ], plugins: 'a' },
```

Comment:
Was this a bug you just fixed? 🙂

Reply:
Technically yes? The `editor.settings` had the wrong value for `forced_plugins` but it still worked as intended for the `plugins` setting.